app/models/stop_area_referential.rb

```diff
@@ -1,4 +1,7 @@
 class StopAreaReferential < ApplicationModel
+  @@keep_syncs = 40
```

Comment:
Tu as 2 fois le même code => fais un concern
Evite aussi les variables de classe: http://www.railstips.org/blog/archives/2006/11/18/class-and-instance-variables-in-ruby/